streaming/text/convert/enwiki/mds/make_train_parallel.py

```diff
@@ -29,7 +29,7 @@ def parse_args() -> Namespace:
     args.add_argument('--out_root',
                       type=str,
                       required=True,
-                      help='Output root directory containing shard dirs named like group-###.')
+                      help='Output directory containing shard dirs named like group-###.')
```

Comment:
but this is a root?

Reply:
Fixed in https://github.com/mosaicml/streaming/pull/171/commits/ad49770b6c7f35b3e143cdf0fc6f5f699f7da917.